test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs

```diff
@@ -542,11 +542,8 @@ 2. D
 
   [c]: c",
             @"<ul>
-<li><p>[a]: a</p>
-</li>
-<li><p>[b]: b</p>
-<p>[c]: c</p>
-</li>
+<li></li>
+<li></li>
```

Comment:
oops, missing one ` *` when testing --

---
In reply to: [107327425](https://github.com/dotnet/docfx/pull/1455#discussion_r107327425) [](ancestors = 107327425)